### ISSUE TITLE
[Question] How to install two Chromium builds ?

### ISSUE THREAD

**Issue by uBlock-user** [henrypp/chromium#15]
I'm already running LKGR version of yours NIK, and I would like to install the stable version along too. But I cannot as Chromium detects version 57 > 55 and kills the installation.

So it would be nice if you offer an option to install Chromium builds on separate drives, for example, I would like to install stable on drive D and LKGR on C. Can you do that ?

**Comment by chrme** [comment]
I only compile the original sources and they doesn't have this option, and i'm not a coder.
I think for this problem the chrlauncher is the best choice :)

**Comment by uBlock-user** [comment]
How is chrlauncher going to help me install Chromium in D drive ?

**Comment by chrme** [comment]
chromium installer are fixed to %appdata% path and doesn't allow more that one version at system, it can be bypassed by portable tools.

**Comment by uBlock-user** [comment]
Multiple chromium installations of portable tools get corrupted, if you' run chrome.exe from the portable tool, instead of chrlauncher.exe. You can reproduce it.

**Comment by henrypp** [comment]
@uBlock-user 
chromium uses lock state for used profile to avoid race condition.
mulptiple chromium installation may work only with different profiles AT SAME TIME.
chrlauncher runs chromium with predefined arguments, but you can define it himself "--user-data-dir=%FULL OR RELATIVE PROFILE PATH%"

**Comment by uBlock-user** [comment]
@henrypp Thanks, but the issue lies when running Chromium 55 from taskbar. 

Here, do this. 

- Install Chromium 55 and Chromium 57. 57 via installer in C drive and install 55 via chrlauncher on D/E drive.

- Pin both of their icons on Windows taskbar.

- Now click on the pinned chromium 55 icon from the taskbar, it will launch chromium 55 with 57's profile and hence another window will open from chromium 57 and error window from Chromium itself saying "something went wrong". 

So basically if you pin 55 and 57 on Windows taskbar, this screw up happens. So I can only pin either of them. I guessing this is a Chromium bug.
